core/core_bind.cpp

```diff
@@ -1332,6 +1339,80 @@ namespace special {
 
 ////// ClassDB //////
 
+bool StaticClassMethodCallableCustom::_equal_func(const CallableCustom *p_a, const CallableCustom *p_b) {
+	if (p_a == p_b) {
+		return true;
+	}
+	if (p_a == nullptr or p_b == nullptr) {
```

Comment:
```suggestion
	if (p_a == nullptr || p_b == nullptr) {
```
No aliases for logical operators